async/asyncamd-tests.ts

```diff
@@ -1,3 +1,5 @@
+/// <reference path="./async.d.ts" />

```

Comment:
no, I think this is breaking change in TS 1.6.
I think 1.6 is correct.
`import async = require("async");` should not lookup from `"./async"`.

```
$  cat a.js
require("b").hello();
$ cat  b.js
exports.hello = function() { console.log("hi!"); };
$ node a.js
module.js:334
    throw err;
          ^
Error: Cannot find module 'b'
    at Function.Module._resolveFilename (module.js:332:15)
    at Function.Module._load (module.js:282:25)
    at Module.require (module.js:361:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/private/tmp/test/a.js:1:63)
    at Module._compile (module.js:426:26)
    at Object.Module._extensions..js (module.js:444:10)
    at Module.load (module.js:351:32)
    at Function.Module._load (module.js:306:12)
    at Function.Module.runMain (module.js:467:10)
```